### ISSUE TITLE
"git_repository" can't setup project when project name not lowercase

### ISSUE THREAD

**Issue by mhdramadhanarvin** [vercel/terraform-provider-vercel#76]
I have problem when setup vercel using terraform.

Condition:
my repository path in gitlab **`user-name/repository-project`** and project name in gitlab **`Repository Project`**, when i'm run **`terraform plan`** its doesn't have any problem, but when run **`terraform apply`** a problem message like this:
```
Plan: 1 to add, 0 to change, 1 to destroy.
vercel_project.example: Destroying... [id=prj_KAhhyBtevEDJUxxxxxxx]
vercel_project.example: Destruction complete after 1s
vercel_project.example: Creating...
╷
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to vercel_project.example, provider "provider[\"registry.terraform.io/vercel/vercel\"]" produced an unexpected new value: .git_repository.repo: was
│ cty.StringVal("user-name/repository-name"), but now cty.StringVal("user-name/Repository Name").
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
```

I think the provider doesn't support when the format path in the repository is not same as project name in the repository or format only supports lowercase without whitespace, but when I change the project name only lowercase without whitespace and same with path in the repository it works.

This is terraform configuration file:
```
resource "vercel_project" "example" {
  name = "project"
  framework = "nextjs"

  git_repository = {
    repo = "user-name/repository-name"
    type = "gitlab"
  }
} 
```

**Comment by dglsparsons** [comment]
Hey @mhdramadhanarvin, thanks for raising this. It looks like a bug on our side - I'll get looking into it 🙌

**Comment by dglsparsons** [comment]
Hi @mhdramadhanarvin, this should be fixed in [v0.10.0](https://registry.terraform.io/providers/vercel/vercel/0.10.0). Please give it a go and let me know if the issue persists!
